backend/src/test/java/com/redhat/cloud/notifications/routers/handlers/event/EventResourceTest.java

```diff
@@ -145,6 +148,17 @@ public class EventResourceTest extends DbIsolatedTest {
     @InjectSpy
     KesselAuthorization kesselAuthorization;
 
+    @BeforeEach
+    void setUp() {
+        // Since we are mocking the back end configuration, we need to set the
+        // following default values for the RBAC calls not to fail. Otherwise
+        // the tests complain because "the initial backoff and the max back off
+        // value must be greater than zero".
+        Mockito.when(this.backendConfig.isRBACEnabled()).thenReturn(true);
+        Mockito.when(this.backendConfig.getRbacRetriesInitialBackOff()).thenReturn(Duration.ofSeconds(1));
+        Mockito.when(this.backendConfig.getRbacRetriesBackOffMaxValue()).thenReturn(Duration.ofSeconds(1));
+    }
```

Comment:
Since I moved some of the settings to the `BackendConfig`, this forced me to mock those settings in some test classes. I was unsure if it was better to have those settings in the `BackendConfig` for consistency even if it forced us to do this, or to just leave those settings where they were. Let me know what you think, please!